server/server.js

```diff
@@ -9,6 +9,7 @@ const MongoStore = require('connect-mongo')(session)
 const passport = require('./passport');
 // route requires
 const user = require('./routes/user')
+require('dotenv').config()
```

Comment:
Move to the very top of the file so that the `.env` file is the first thing loaded.